app/models/conversion_host/configurations.rb

```diff
@@ -113,12 +114,13 @@ def disable_queue(auth_user = nil)
 
   def disable(_params = nil, _auth_user = nil)
     resource_info = "type=#{resource.class.name} id=#{resource.id}"
-    _log.debug("Disabling a conversion_host #{resource_info}")
+    raise "There are active migration tasks running on this conversion host" if active_tasks.present?
 
+    _log.debug("Disabling a conversion_host #{resource_info}")
     disable_conversion_host_role
     destroy!
   rescue StandardError => error
-    raise
+    raise error
```

Comment:
Why rescue the error then re-raise it without doing anything else?

Reply:
(Side note, `raise` and `raise error` will do the same thing in a `rescue => error` section)

Reply:
Because otherwise the `error` variable is not available to `ensure` block.

Reply:
Ah didn't see you using that down there :+1: